src/test/Components/CreateImageWizard/steps/ImageOutput/ImageOutput.test.tsx

```diff
@@ -265,44 +259,6 @@ describe('Step Image output', () => {
     );
   });
 
-  test('VMware checkbox select and unselect works', async () => {
```

Comment:
This tested if switching between the radios and toggling parent category works correctly. Since we have simple checkboxes now it's no longer needed.